types/instrumentation.d.ts

```diff
@@ -79,3 +80,19 @@ export interface IClarityActivateErrorState extends IInstrumentationEventState {
 export interface ITriggerState extends IInstrumentationEventState {
   key: string;
 }
+
+export interface IPerformanceState extends IInstrumentationEventState {
+  procedure: string;
+  duration: number;
```

Comment:
>duration [](start = 2, length = 8)

might be nice to also include start time so we have a full picture

Reply:
I don't really see how that would be useful

---
In reply to: [261791885](https://github.com/Microsoft/clarity-js/pull/139#discussion_r261791885) [](ancestors = 261791885)